<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel użytkownika</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string>font-size: 16px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Witaj!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bmiButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kalkulator BMI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addMedButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dodaj lek</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="historyButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Historia leków</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Eksportuj dane</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="motivateButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Motywacja</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="logoutButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wyloguj się</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
